pkg/proc/gdbserial/gdbserver.go

```diff
@@ -547,6 +547,14 @@ func LLDBLaunch(cmd []string, wd string, flags proc.LaunchFlags, debugInfoDirs [
 
 	if runtime.GOOS == "darwin" {
 		process.Env = proc.DisableAsyncPreemptEnv()
+		// filter out DYLD_* environment variables
```

Comment:
Explain why here.

Reply:
To have all the darwin env changes in the same spot - or you want me to add a comment saying that?

Reply:
I think this comment should explain why we are filtering out DYLD_ variables.
Basically it should repeat what the commit message says and both should be more precise about what changed and when it did than what the current commit message is, if possible.

Reply:
Understood - I'll amend in few minutes.

Reply:
I hope it is more clear now.